bin/restore.sh

```diff
@@ -44,17 +44,21 @@ fi
 echo "expands ${TARGET}..."
 time ${TAR_CMD} ${TAR_OPTS} ${TARBALL_FULLPATH} -C ${DIRNAME} ${BASENAME}
 
-
 # restore database
 if [ "x${MONGODB_DBNAME}" != "x" ]; then
-  MONGORESTORE_OPTS="${MONGORESTORE_OPTS} -d ${MONGODB_DBNAME}"
-  TARGET=${TARGET}/${MONGODB_DBNAME}
-fi
-if [ "x${MONGODB_USERNAME}" != "x" ]; then
-  MONGORESTORE_OPTS="${MONGORESTORE_OPTS} -u ${MONGODB_USERNAME} -p ${MONGODB_PASSWORD}"
+  MONGORESTORE_OPTS="--nsInclude=${MONGODB_DBNAME}.* ${MONGORESTORE_OPTS}"
```

Comment:
When specified `MONGODB_DBNAME`, use `--nsInclude` option instead of `-d` option.
(`-d` option is deprecated, described in [here](https://docs.mongodb.com/manual/reference/program/mongorestore/#cmdoption-mongorestore-db))